mne/preprocessing/tests/test_maxwell.py

```diff
@@ -25,11 +25,24 @@
                         'test_move_anon_raw_simp_stdOrigin_sss.fif')
 sss_nonstd_fname = op.join(data_path, 'SSS',
                            'test_move_anon_raw_simp_nonStdOrigin_sss.fif')
+sss_fine_cal_fname = op.join(data_path, 'SSS',
+                             'test_move_anon_raw_fineCal_sss.fif')
+
+fine_cal_fname = op.join(data_path, 'SSS', 'sss_cal_3053.dat')
+fine_cal_fname_3d = op.join(data_path, 'SSS', 'sss_cal_3053_3d.dat')
+
+int_order, ext_order = 8, 3
```

Comment:
I use these vars to test some of the more fundamental functions that don't use these values as defaults. (See lines 71-76)